src/Project/WorkJobs.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Job} from './Job';
+
+export class WorkJobs extends Array<Job> {}
```

Comment:
I'm not sure but when it comes to alias, this seems be normally used.
```suggestion
export type WorkJobs = Array<Job> {};
```

Reply:
I too am not sure and don't know indepth about `type` in TS, I would just like to go with class extends for now...